qa/workunits/fs/misc/direct_io.py

```diff
@@ -34,14 +36,15 @@ def main():
         tmpf.write('2')
 
     proc = subprocess.Popen(['rados', '-p', pool_name, 'put', obj_name, 'tmpfile'])
-    proc.wait();
+    proc.wait()
 
     os.lseek(fd, 0, os.SEEK_SET)
-    out = os.read(fd, 1);
+    out = os.read(fd, 1)
     if out != '2':
         raise RuntimeError("data were not directly read from object store")
 
-    os.close(fd);
-    print 'ok'
+    os.close(fd)
+    print('ok')
```

Comment:
oic, `print()` is introduced in py2.6, and we've bumped the supported python version to >= 2.6